### ISSUE TITLE
[eslint-plugin] `valid-styles` mistakenly reports multi-value uses of `backgroundBlendMode` as invalid

### ISSUE THREAD

**Issue by dwei-figma** [facebook/stylex#1213]
### Describe the issue

When there are multiple values for `background`, multiple values can be provided to [`background-blend-mode`](https://developer.mozilla.org/en-US/docs/Web/CSS/background-blend-mode) to apply a different blend mode per background. 

``` ts
tintedContainer: (color: string) => ({
  background: `linear-gradient(rgba(245, 245, 245, 0.6), rgba(245, 245, 245, 0.6)),
               linear-gradient(#F5F5F5, #F5F5F5),
               linear-gradient(${color}, ${color})`,
  // eslint-disable-next-line @stylexjs/valid-styles
  backgroundBlendMode: 'color, luminosity, normal',
}),
```

### Expected behavior

Multiple values to `backgroundBlendMode` should not be reported as an error.

### Steps to reproduce

1. Have the `@stylexjs/valid-styles` ESLint rule enabled.
2. Write a StyleX style with the `backgroundBlendMode` rule set to multiple comma-separated values.
3. Observe complaint from ESLint.

**Comment by mellyeliu** [comment]
Thanks for reporting! There's quite a few cases of the `valid-styles` rule being too permissive/too strict on various properties and we're looking to do an overhaul soon with some external tooling

**Comment by dwrth** [comment]
I'd like to help out with this one as well!
